src/components/Share/Share.js

```diff
@@ -51,7 +51,7 @@ const Share = () => {
     <div className="wkp-share">
       <SharePermalinkButton />
       {renderConf(config[0], t)}
-      <CanvasSaveButton title={t('Karte als Bild speichern')} map={map}>
+      <CanvasSaveButton aria-label={t('Karte als Bild speichern')} map={map}>
```

Comment:
No more title ?

Reply:
React doesn't complain about aria-label ? should not be ariaLabel ?

Reply:
Nop It complains about ariaLabel (taken as a prop and not as the aria-label of the element) :)

Reply:
I add the title to the children (title prop has been removed from the component: https://github.com/geops/react-spatial/commit/f49dbfe9f9e9063c77d761fc9e96ac45a59144a9#diff-84392269f1dcee3ac19b57cd31741243)